apps/condo/domains/organization/utils/serverSchema/guards.js

```diff
@@ -1,4 +1,4 @@
-const { findOrganizationEmployee } = require('../../../../utils/serverSchema/Organization')
+const { findOrganizationEmployee } = require('@condo/domains/organization/utils/serverSchema/Organization')
```

Comment:
`findOrganizationEmployee` should be replaced by: `OrganizationEmployee.getAll(condext, .....)`
it's legacy